spec/test_app/db/schema.rb

```diff
@@ -70,6 +70,7 @@
     t.datetime "published_at"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "banner_url"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.